tensorflow/core/ops/ops.pbtxt

```diff
@@ -10313,6 +10313,154 @@ op {
   summary: "Computes gradients of the maxpooling function."
 }
 op {
+  name: "MaxPoolGradGrad"
```

Comment:
Oh, you probably need to read this: https://github.com/tensorflow/tensorflow/blob/754048a0453a04a761e112ae5d99c149eb9910dd/tensorflow/g3doc/how_tos/adding_an_op/index.md#backwards-compatibility

My suggestion would be to declare this as a new type Tinput with a default of DT_FLOAT to preserve backwards compatibility as in the suggestion in the link above.  It sucks to have to declare a new type for this, but I can't think of alternative that is backwards compatible.